politeiawww/backend.go

```diff
@@ -2065,7 +2053,7 @@ func (b *backend) ProcessComment(c www.NewComment, user *database.User) (*www.Ne
 		return nil, fmt.Errorf("ProcessComment: getBestBlock: %v", err)
 	}
 
-	if getVoteStatus(ir, bb) == www.PropVoteStatusFinished {
+	if getVoteStatus(&ir, bb) == www.PropVoteStatusFinished {
```

Comment:
I don't see a change to the `getVoteStatus` function definition, is this correct?